util/java/tmxviewer-java/tmxviewer-java.qbs

```diff
@@ -0,0 +1,15 @@
+import qbs
+
+JavaJarFile {
+    Depends { name: "libtiled-java" }
+
+    entryPoint: "TMXViewer"
+
+    files: ["src/**/*.java"]
+
+    Group {
+        fileTagsFilter: ["java.jar"]
+        qbs.install: true
+        qbs.installDir: "jar"
+    }
```

Comment:
The Java-based TMXViewer application is just a small example and the Qt-based tmxviewer is much more suitable to ship as basic TMX viewer (and it already installed). I don't think this one should install anywhere.